sdk/JavaScript/TestFramework/js/tests/pushTests.js

```diff
@@ -5,25 +5,26 @@
 /// <reference path="../testFramework.js" />
 
 function definePushTestsNamespace() {
-    var tests = [];
-    var i;
-    var tableName = 'w8PushTest';
-    var toastTemplateName = 'newsToastTemplate';
-    var tileTemplateName = 'newsTileTemplate';
-    var badgeTemplateName = 'newsBadgeTemplate';
-    var rawTemplateName = 'newsRawTemplate';
-    var templateNotification = {
-        "News_English": "World News in English!",
-        "News_French": "Nouvelles du monde en français!",
-        "News_Mandarin": "在普通话的世界新闻 ！",
-        "News_Badge": "10"
-    }
+    var tests = [],
+        channelUri,
+        registrationWaitInterval = 10000;
+
+    var testTemplate = {
+            body: '<toast><visual><binding template="ToastText01"><text id="1">$(message)</text></binding></visual></toast>',
+            name: 'templateForToastWinJS',
+            headers: {
+                'X-WNS-Type': 'wns/toast',
+                'X-WNS-TTL': '1000'
+            },
+            lowerHeaders: {
+                'x-wns-type': 'wns/toast',
+                'x-wns-ttl': '1000'
+            },
```

Comment:
Why two representations of the same thing? Pretty sure header names are not case sensitive and we can get away with just using the lower case representation.